### PR TITLE
Run tests locally

### PR DESCRIPTION
**Pull request by ewansheldon**
## Describe your changes
Remove forked mainnet configuration, and run all tests locally. all external contracts from uniswap, chainlink etc are mocked

## Checklist before requesting a review
- [X] I have verified that all tests pass
- [X] If the public API has changed, affecting the IBCO interaction, I have expressed it clearly above
- [X] If the ABI has changed, affecting auxiliary functionality, I have expressed it clearly above

### CODE REVIEW

File: test/stage2/bondingEvent.js [hunk 1]
```diff
@@ -469,12 +475,12 @@ describe('BondingEvent', async () => {
       );
       const positions = await BondingEvent.getPositions();
       expect(positions).to.be.length(2);
-      await swap(USDC, SEuro, etherBalances['125K'].mul(5));
 
       await expect(BondingEvent.clearPositionAndBurn(positions[0].tokenId)).to.be.revertedWith('err-no-wallet-assigned');
 
       await BondingEvent.setExcessCollateralWallet(wallet.address);
 
+      // const collect = BondingEvent.clearPositionAndBurn(positions[0].tokenId);
```
Comment (therealdannzor) [inline test/stage2/bondingEvent.js:483]: dead comment

**Review by therealdannzor — APPROVED** [review]
Nice optimizations 😃

File: test/stage2/bondingEvent.js (hunk 1 above)
Comment (therealdannzor) [reply test/stage2/bondingEvent.js:483]: sure, you can push to master 👍

Comment (ewansheldon) [reply test/stage2/bondingEvent.js:483]: can i? sadly i don't think i can 😢

Comment (therealdannzor) [reply test/stage2/bondingEvent.js:483]: no, I disabled that for all of us. next time you open a PR you can remove it, or if you see some other cleanup to do you can open a PR with the fix too 😄

**Pull request merged**